internal/coordinator/api/api.go

```diff
@@ -537,6 +537,7 @@ func (s *API) getTaskLog(c echo.Context) error {
 		si = "25"
 	}
 	size, err := strconv.Atoi(si)
+	q := c.QueryParam("q")
```

Comment:
nit: there's a swaggo docblock for this function, would you mind adding this new `@Param` there?